doc_src/fish_indent.txt

```diff
@@ -7,12 +7,16 @@ fish_indent [OPTIONS]
 
 \subsection fish_indent-description Description
 
-`fish_indent` is used to indent a piece of fish code. `fish_indent` reads commands from standard input and outputs them to standard output.
+`fish_indent` is used to indent a piece of fish code. `fish_indent` reads commands from standard input and outputs them to standard output or a specified file.
 
 The following options are available:
 
 - `-d` or `--dump` dumps information about the parsed fish commands to stderr
 
+- `-w` or `--write` indents a specified file and immediatly writes to that file
```

Comment:
Typo: immediatly => immediately

Also, the `-W` option isn't necessary or useful. Anyone who wants to send the output to a specific file should use the usual shell redirection; e.g., `fish_indent script.fish > new_script.fish`.